README.md

```diff
@@ -2,22 +2,31 @@
 
 You can deploy 1Password SCIM Bridge on any supported infrastructure that allows ingress by your identity provider and egress to 1Password servers. Here you'll find configuration files and best practices to help with your deployment. Learn more about [automating 1Password provisioning with SCIM](https://support.1password.com/scim/).
 
-## Automatic (one-click) deployment
+## Marketplace apps
 
-The easiest way to deploy 1Password SCIM Bridge is with our one-click apps, which are currently available for Google Cloud Platform and DigitalOcean:
+If you'd prefer to deploy 1Password SCIM Bridge using a marketplace, you can use Google Cloud Platform or DigitalOcean:
```

Comment:
I just spoke with @ag-adampike on a call and we came up with a slightly modified version of this phrasing, since the context and audience are different from AKB. @black-bryan how does this sound?

```suggestion
For an automated deployment of 1Password SCIM Bridge where some settings are pre-configured for you, use a marketplace app from Google Cloud Platform or DigitalOcean:
```